cli/magefile.go

```diff
@@ -43,6 +45,8 @@ func BuildApi() error {
 	}
 	if err := shellcmd.RunAll(
 		shellcmd.Command("CGO_ENABLED=0 go build -o symphony-api"),
+		shellcmd.Command("CGO_ENABLED=0 GOARCH=arm64 go build -o symphony-api-arm64"),
```

Comment:
Do you want to upload these two packages to release page? Then please modify release.yml github action accordingly.

Reply:
Good catch. Added.